WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFragment.kt

```diff
@@ -139,6 +143,7 @@ class QuickStartFragment : Fragment() {
                 .setPositiveButton(getString(R.string.quick_start_button_skip_positive)) { _, _ ->
                     (view as ScrollView).smoothScrollTo(0, 0)
                     viewModel.skipAllTasks()
+                    AnalyticsTracker.track(QUICK_START_LIST_SKIP_ALL_TAPPED)
```

Comment:
Should the `QUICK_START_LIST_SKIP_ALL_TAPPED` statistic be tracked when the ***Skip All*** button is tapped [here](https://github.com/wordpress-mobile/WordPress-Android/pull/8228/files#diff-567e5fd8f7fa306b7c6decb10360d595L113) rather than when the positive button is tapped in the confirmation dialog?

Reply:
Yes, I asked @SylvesterWilmott about it, and he confirmed that tracking the dialog confirmation is ok.